drivers/hmis/spec/models/hmis/form/form_processor_spec.rb

```diff
@@ -2004,6 +2004,19 @@ def process_assessment(new_hud_values: nil)
       expect(cls.information_date).to eq(information_date)
     end
 
+    it 'should work when verified by project ID is provided' do
+      hud_values = {
```

Comment:
could use `merge` with the `hud_values` defined for this block, so it's clear what's changing